assets/prometheus-k8s/prometheus.yaml

```diff
@@ -98,12 +98,6 @@ spec:
   - kube-rbac-proxy
   securityContext: {}
   serviceAccountName: prometheus-k8s
-  serviceMonitorNamespaceSelector:
-    matchExpressions:
-    - key: openshift.io/cluster-monitoring
-      operator: Exists
-  serviceMonitorSelector:
-    matchExpressions:
-    - key: k8s-app
-      operator: Exists
+  serviceMonitorNamespaceSelector: {}
```

Comment:
In case I would like to have parts of my namespace monitored by the `prometheus-k8s` Prometheus, and other parts monitored by my own Prometheus, how do I prevent the `prometheus-k8s` to pick up the ServiceMonitors of the second part?

Reply:
As we are limiting the namespaces to monitor the “all” selection here is restricted to “all” core openshift component namespaces.